### ISSUE TITLE
`restart_process` having issues with base image `ubi-minimal`

### ISSUE THREAD

**Issue by henry-harness** [tilt-dev/tilt-extensions#432]
Originally, my image was created using `alpine` and I needed to switch to `ubi-minimal`. The conversion was successful and was able to create my image. I am able to start the image in tilt too. But when i make a change to the src, the restart is not successful. It gets into a funky state where the app is trying bind to a port, but it is unable to.

**Comment by nicks** [comment]
I don't think `ubi-minimal` has `tar` in it?

Lots of dev tooling simply stops working if you try to use it with a container that doesn't have `tar`, for example: https://github.com/kubernetes/kubernetes/issues/58512

**Comment by henry-harness** [comment]
I installed `tar` and it is the same issue.

**Comment by henry-harness** [comment]
I changed my image to go back to `alpine`, and noticed i'm getting the same issue. So i don't think specific to `UBI` base image issue. I did recently updated tilt. But i don't think that is an issue 🤔

**Comment by henry-harness** [comment]
Trying to debug this a little, it seems like the original app is not being killed/stopped.

I went into the pod, and executed the `tilt-restart-wrapper` command that is shown in `ps aux`. Then I see a new instance of tilt-restart-wrapper, entr, and the app. So double of everything. When i manually changed `tmp/.restart-proc`, the second app PID does get removed, and a new app is started. This PID is different from the second PID.

Hopefully this makes sense.

Other observations - I've noticed that `entr` is not being the latest, and so the `-z` parameter is not known. entr still runs though. Is it possible to update `entr` and so the `-z` is used?
